src/self_test/self_test.c

```diff
@@ -163,6 +179,39 @@ static int self_test_init(char *dest_name, crt_context_t *crt_ctx,
 		return -DER_MISC;
 	}
 
+	if (is_nopmix) {
```

Comment:
shouldnt we be using tc_cli_start_basic() instead?

Reply:
That would be ideal, but selftest can also serve as a server which requires the CRT_FLAG_BIT_SERVER.
It will take some changes in tc_cli_start_basic to handle this situation, and might impact other generic client test using this function.
My thought is to minimize changes and keep the handling inside of selftest.